allauth/templates/tests/test_403_csrf.html

```diff
@@ -0,0 +1,86 @@
+{% load socialaccount %}
+<!DOCTYPE html>
```

Comment:
I don't think there is a need to have a full template in here. It could be a minimal template with just  the provider_login_url tag in there.